refbox/src/refBox.js

```diff
@@ -237,8 +247,11 @@ class RefBox extends React.Component {
     })
   }
 
-  updateScoreTable(data) {
-    this.setState({scoreTable: data});
+  updateChallengeInfo(data) {
+    this.setState({
+      challengeDescription: data.description,
+      scoreTable: data.score_table,
+    })
```

Comment:
Will this and similar repeated functions be pulled out into a utils.js module to reuse in both refBox and audienceInfo?

Reply:
Yes, see #6